### PR TITLE
Fibonacci in Groovy

### PR DESCRIPTION
**Pull request by LoicB**
Congrats on taking the first step to contributing to the [Sample Programs in Every Language][1] repository!

For simplicity, please make sure that your pull request includes one and only one sample program and language.

## Please Complete the Following

- [ ] I would **NOT** like credit for my contribution in the article (credit will be included otherwise)
- [ ] I plan to write the article for this code snippet
- [x] I named the pull request using `<Sample Program> in <Language>` format
- [x] I created/updated the language README
  - [x] I added the sample program name to the README
  - [x] I added fun facts (i.e. debut, developer, typing, etc.)
  - [ ] I added reference link(s) to the README
  - [ ] I added solution citations when necessary (see [plagiarism][2])
- [ ] I fixed #your-issue-number-here

## Notes

Feel free to put whatever you want here.

[1]: https://therenegadecoder.com/code/sample-programs-in-every-language/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/CONTRIBUTING.md#plagiarism

### CODE REVIEW

**Comment by Noah11012** [comment]
Could you change it where the user can generate at least 93 fibonacci numbers?

**Comment by LoicB** [comment]
I modified the code in order to work with higher numbers.

The warning is due to an issue with Groovy and the SDK 9.
https://issues.apache.org/jira/browse/GROOVY-8339

**Comment by jrg94** [comment]
Fair enough! I'm happy with the code now, but for completeness we should probably chat about value limits. What's the largest number Groovy can produce, and can you limit your program to only accept numbers up to that limit?

File: archive/g/groovy/Fibonacci.groovy [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Fibonacci {
+    static void main(String[] args) {
+        if (args.length < 1 || !args[0].isInteger()) {
+            println 'please provide an integer between 1 and 100'
```
Comment (jrg94) [inline archive/g/groovy/Fibonacci.groovy]: Based on discussion, let's just change this line to say "please provide an integer". I don't think we need to set limits.

**Pull request merged**